packages/web/src/views/launchpad/components/launchpad-active-projects/launchpad-active-projects-header/LaunchpadActiveProjectsHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  ActiveProjectsHeaderTextWrapper,
+  ActiveProjectsWrapper,
+} from "./LaunchpadActiveProjectsHeader.styles";
+
+export interface LaunchpadActiveProjectsHeaderProps {
+  count: number | "-";
```

Comment:
If there is no data, treating it as null seems like a good idea for comparisons.
Why not handle the string handling of ‘-’ at showtime?
```javascript
const countStr = useMemo(()=> {
    if (count === null) {
        return "-"
    }

    return count.toLocaleString()
}, [count])
```